contrib/opentimelineio_contrib/adapters/tests/test_aaf_adapter.py

```diff
@@ -974,7 +982,7 @@ def test_aaf_sourcemob_usage(self):
         Each clip stores it's source mob usage AAF value as metadata in`SourceMobUsage`.
         For sub-clips this value should be `Usage_SubClip`.
         """
-
+        print(otio.__file__)
```

Comment:
Is this print statement intentional, or left over from debugging?

Reply:
Good catch. That's definitely just a debug print!